src/main/java/com/bouncestorage/chaoshttpproxy/ChaosConfig.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2015 Bounce Storage, Inc. <info@bouncestorage.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.bouncestorage.chaoshttpproxy;
+
+import java.util.List;
+import java.util.Properties;
+
+public interface ChaosConfig {
```

Comment:
Instead of having a bunch of stream handling in `DefaultChaosConfig`, would it be better to have a single implementation which has a `DefaultChaosConfig(Properties)` constructor?